packages/wdio-config/src/node/ConfigParser.ts

```diff
@@ -228,6 +238,24 @@ export default class ConfigParser {
         }
     }
 
+    /**
+     * merge only reporters and services to the config object
+     * @param object desired object to merge into the config object
+     * @private
+     */
+    private mergeReportersAndServices(object: MergeConfig) {
```

Comment:
Do we still need this function?

Reply:
Yes because any way we have to merge config which is added via launcher